lib/std/build.zig

```diff
@@ -1559,6 +1560,11 @@ pub const LibExeObjStep = struct {
         self.filter = text;
     }
 
+    pub fn setUseTestEventedIO(self: *LibExeObjStep, use_evented_io: bool) void {
```

Comment:
Minor: this should be `setUseTestEventedIo` per the style guide: https://ziglang.org/documentation/master/#Names

Reply:
Fixed. Thanks